test_autolens/interferometer/test_fit_interferometer.py

```diff
@@ -204,9 +211,12 @@ def test__fit_figure_of_merit__include_hyper_methods(interferometer_7):
 
     galaxy_light = al.Galaxy(redshift=0.5, bulge=al.lp.EllSersic(intensity=1.0))
 
-    pix = al.pix.Rectangular(shape=(3, 3))
-    reg = al.reg.Constant(coefficient=1.0)
-    galaxy_pix = al.Galaxy(redshift=1.0, pixelization=pix, regularization=reg)
+    pixelization = al.Pixelization(

```

Comment:
There's so much repetition I guarantee you can make these tests way more concise and easy to manage using fixtures and/or parametrize

Reply:
It might be worth doing some pair programming so we can go through an example